robolectric/src/test/java/org/robolectric/shadows/ShadowActivityTest.java

```diff
@@ -1017,6 +1019,117 @@ public void getPendingTransitionExitAnimationResourceId_should() {
     assertThat(shadowOf(activity).getPendingTransitionExitAnimationResourceId()).isEqualTo(2);
   }
 
+  @Test
+  @Config(minSdk = UPSIDE_DOWN_CAKE)
+  public void getOverriddenActivityTransitionOpen_withoutBackgroundColor() {
+    ActivityController<Activity> controller = buildActivity(Activity.class).setup();
```

Comment:
That's a good nitpick, actually. Used try-with-resources to avoid explicit `destroy` call and to ensure the activity is destroyed even if a test fails.